theme/templates/pages/release_roadmap.html

```diff
@@ -0,0 +1,149 @@
+{% extends "base.html" %}
+
+{% block title %}{% trans %}Previsional Release Roadmap{% endtrans %}{% endblock %}
+
+{% block extra_head %}
+<link rel="stylesheet" href="{{ SITEURL }}/theme/css/pages/release_roadmap.css">
+
+<script src="https://cdn.jsdelivr.net/npm/mermaid@11.2.1/dist/mermaid.min.js" integrity="sha384-7J/UM6Vde6cXqst3EI86P36213Jz5rooW1EEUvweQ3F3gQOHkTuSKAX67ANmYljn" crossorigin="anonymous"></script>
```

Comment:
not a fan of having rendering the static content client side all the time. I'd be nice if there was a pelican plugin that rendered this inline at build time... Wrapping https://github.com/ouhammmourachid/mermaid-py may be feasible. We already have other plugins (such as the privacy-friendly youtube embed) that could be used as an example.

Reply:
Unfortunatelly, this Python plugin doesn't support Timeline and Gitgraph. However, I agree it is definitely sub-optimal to address that on client side. One solution could be to use the (Node-based) mermaid-cli which allows you to generate SVG out of the diagram, so this could be added in a `Makefile` for example and/or the CI. WDYT?

Reply:
yeah neither are great. I'd almost prefer the current client side solution then I guess.